pkgs/servers/web-apps/sftpgo/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "sftpgo";
+  version = "2.4.0";
+
+  src = fetchFromGitHub {
+    owner = "drakkan";
+    repo = "sftpgo";
+    rev = "v${version}";
+    sha256 = "sha256-A4+YmChUPn+6P0rBuzYcABXyjXRZWY5KS1YcFZHCrYo=";
+  };
+
+  vendorSha256 = "sha256-kwluXCkbclrfRsrdqSxb5+TCBpVPZmDmrbpzR+yuQdQ=";
+  subPackages = [ "." ];
+
+  postInstall = ''
+    cp -r ./{openapi,static,templates} $out/
```

Comment:
We shouldn't put those at ``/``. Maybe $out/share/sftpgo/* would be a better place?

Reply:
Changed to `$out/share/sftpgo/`